lms/data_tasks/report/create_from_scratch/02_entities/01_users/03_users_sensitive/01_create_view.sql

```diff
@@ -0,0 +1,30 @@
+DROP MATERIALIZED VIEW IF EXISTS report.users_sensitive CASCADE;
```

Comment:
The thinking with the name here is that we have have `<x>` and `<x>_sensitive` to remind us to treat these carefully.